resources/css/components/_expander.scss

```diff
@@ -1,6 +1,10 @@
+.expander {
+    position: relative;
+}
+
 .expander button {
     align-items: center;
-    background-color: var(--ll-background-color);
+    background-color: transparent;
```

Comment:
If/when you integrate UIO, you'll need to remember to account for this when adding the contrast themes.

I also wonder if you could have also achieved the same interaction by setting the `pointer-events: none` on the badge. In a quick trial both seemed to behave similarly. However, there may be other issues that I didn't consider.